### DISCUSSION TITLE
ScummVM version out of date

### DISCUSSION THREAD

**Discussion by IanMoyes** [microsoft/winget-pkgs#87687] [General] [answered]
2.6.1 is the most recent version and can be found at [https://downloads.scummvm.org/frs/scummvm/2.6.1/scummvm-2.6.1-win32.exe](url) - can someone amend ScummVM.ScummVM to point at this?

Many thanks!

**Comment by mdanish-kh** [comment, marked as answer]
Will be resolved once the following PR gets merged. 
- https://github.com/microsoft/winget-pkgs/pull/87645/

For future package related problems, it is recommended to [open an issue](https://github.com/microsoft/winget-pkgs/issues/new/choose) so that it can be tracked more easily.

**Reply by IanMoyes** [reply]
Thanks - duly noted! I searched discussions but didn't think of looking in the issues tracker.
